keps/sig-auth/20190607-certificates-api.md

```diff
@@ -335,18 +392,19 @@ type CertificateSigningRequestCondition struct {
 
 type RequestConditionType string
 
-// These are the possible conditions for a certificate request.
+// These are well-known conditions types for a certificate request.
```

Comment:
I assume this is it to allow arbitrary conditions for use by custom signers?

Reply:
right